Custom Definitions.event

```diff
@@ -39,6 +46,11 @@
 #define Jazzak 0x42
 #define Dennis 0x50
 
+#define Ximena 0xA
+
+#define Grudge 0x4A
+#define Ch5Generic 0x4B
```

Comment:
Please use the generic ID assigned to you for your generics. (In this case, it is 0x8A, and 0x8B if you need two)